gluten-celeborn/src/main/scala/org/apache/spark/shuffle/CelebornColumnarShuffleWriter.scala

```diff
@@ -106,9 +106,6 @@ abstract class CelebornColumnarShuffleWriter[K, V](
       .getOrElse(0)
   }
 
-  protected val bufferCompressThreshold: Int =
-    GlutenConfig.get.columnarShuffleCompressionThreshold
-
```

Comment:
It seems we removed this config, maybe we should also update here?
https://github.com/apache/incubator-gluten/blob/main/gluten-substrait/src/main/scala/org/apache/gluten/config/GlutenConfig.scala#L182